src/app/pages/Admin.tsx

```diff
@@ -107,8 +151,22 @@ export const Admin = () => {
 									rotationDeg: shortestRotation360(rotationDeg + rotation),
 								})
 							}}
-							onPointerDown={() => {
-								setSelectedRobot(mac)
+							onPointerDown={(args) => {
+								startTimer()
```

Comment:
I would like to see a simpler API that hides the implementation detail:

```typescript
onPointerDown={(args) => {
  startLongPressDetection()
}}
onPointerUp={() => {
  if (endLongPressDetection()) {
    console.log(`Long pressed!`)
  } else {
    console.log(`Not long pressed!`)
  }
}}
```

Reply:
I came with this implementation. 

``` JavaScript 

onPointerDown={(args) => {
    startLongPressDetection()
 }}

onPointerUp={() => {
   const isLongPressDetected = endLongPressDetection()
  
   isLongPressDetected
	  ? setSelectedRobot({ mac, action: 'reposition' })
	  : setSelectedRobot({ mac, action: 'rotation' })
}}

```

I think `isLongPressDetected` var add legibility to the code and for that I'm using it instead of using the method directly on the validation.

Reply:
Other option could be to rename `endLongPressDetection` to `isLongPressDetected` instead but maybe it will make the hook API a little bit ambiguous

Reply:
Sounds good to me.